Makefile.am

```diff
@@ -17,6 +17,7 @@ WIN32_BUILD_FILES = \
 EXTRA_DIST = \
 	autogen.sh \
 	scripts/gen-api-gtkdoc.py \
+        geany.appdata.xml.in \
```

Comment:
Sorry, I am probably wrong, but I don't see it. Everything lines up just well:
https://github.com/badshah400/geany/blob/master/Makefile.am